### PR TITLE
TRIVIAL: Fix sdk-ui-filters scss imports

### PR DESCRIPTION
**Pull request by kandl**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/188/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1782/ : SUCCESS in 9m 21s

File: libs/sdk-ui-filters/styles/scss/components/ListItem.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // (C) 2007-2019 GoodData Corporation
+@import "~@gooddata/goodstrap/lib/core/styles/typo-mixins.scss";
```
Comment (no23reason) [inline libs/sdk-ui-filters/styles/scss/components/ListItem.scss:2]: Why was this added?

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/199/ : SUCCESS in 42s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/454/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/1785/ : SUCCESS in 9m 21s
